Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -528,7 +530,7 @@ void MainWindow::load_plugin(QString fileName, bool blacklisted)
       {
         if ( plugin_blacklist.contains(name) ){
           qDebug("### Ignoring plugin \"%s\".", qPrintable(fileName));
-          return;
+          return true;
```

Comment:
Actually the function returns true if fileName leads to a plugin, not if the plugin is loaded. I needed a way to get all plugins in load_plugins, and load_plugin() is the one that filters the files, so I used the return value. using a reference to a bool might be clearer though.